ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py

```diff
@@ -34,60 +35,74 @@ def shell(cmd, debug=False, env_=dict(os.environ)):
         print("Error: %s" % str(ex))
     return result
 
+
 def usage():
-    print ("""
-Usage: python %s [--backup [yes|no]] file1 file2 ...
+    print("""
+Usage: python %s [--backup [yes|no]] [--force [yes|no]] file1 file2 ...
 
 This tool will go through all the test items in the specified job yaml files, call generate-cron-entry.sh to set each job's cron entiries in batch.
 
 By default, --backup is enabled, it will backup the editing job yaml for your comparation before your final submit.
 
-# Presiquisite:
+If you set the --force parameter, it is propagated to the generation bash script, to also consider test configs disabled by default (for example, the *-baremetal-*).
+
+# Prerequisite:
 Download https://raw.githubusercontent.com/openshift/release/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh into $PATH, and make it executable.
 """ % (sys.argv[0]))
 
+
 def get_cli_opts():
     backup_opt = "yes"
-    options, args = getopt.getopt(sys.argv[1:], "hb:", ["help", "backup="])
+    force_generation = False
+    options, args = getopt.getopt(sys.argv[1:], "fhb:", ["force", "help", "backup="])
     for opt, value in options:
         if opt in ("-h", "--help"):
             usage()
         if opt in ("-b", "--backup"):
             backup_opt = value
+        if opt in ("-f", "--force"):
+            force_generation = True
+
+    return backup_opt, force_generation, args
 
-    return backup_opt, args
 
 if __name__ == "__main__":
-    backup_flag, target_file_list = get_cli_opts()
+    backup_flag, force_generation, target_file_list = get_cli_opts()
 
     if len(target_file_list) == 0:
         usage()
         sys.exit(-1)
 
     for target_file in target_file_list:
-        print("Updating %s" %(target_file))
-    
+        print("Updating %s" % (target_file))
+
         with open(target_file, 'r') as file:
             all_data = yaml.safe_load(file)
         file.close()
-    
+
         all_tests_list = all_data['tests']
         new_tests_list = []
+        cron_prefix = "cron:"
```

Comment:
I guess we can move the line out of the for loop, e.g: to line 75.

Reply:
sure